cvat/requirements/base.in

```diff
@@ -8,14 +8,12 @@ clickhouse-connect==0.6.8
 coreapi==2.3.3
 datumaro @ git+https://github.com/cvat-ai/datumaro.git@8a14a99fe17f19d98595a2a4a74ab459051cc23b
 dj-pagination==2.5.0
-dj-rest-auth[with_social]==2.2.7
-
-# dj-rest-auth[with_social] includes django-allauth but with version range: >=0.40.0,<0.53.0
-# This does not suit us in the case when one of the previous allauth version was installed.
 # Despite direct indication allauth in requirements we should keep 'with_social' for dj-rest-auth
 # to avoid possible further versions conflicts (we use registration functionality)
 # https://dj-rest-auth.readthedocs.io/en/latest/installation.html#registration-optional
-django-allauth>=0.52.0
+dj-rest-auth[with_social]>=5.0.2,<6
+# The same django-allauth version range as dj-rest-auth requirements
```

Comment:
I don't understand this comment. We don't need to specify dj-rest-auth's requirements, pip already knows them. We should be setting the constraints according to our own requirements.

Reply:
@Marishka17 Ping?

Reply:
Also, FWIW, this comment will not necessarily remain accurate in the future, since future versions of `dj-rest-auth` may change their dependency specifications.